cpp/python/tests/conftest.py

```diff
@@ -3,7 +3,7 @@
 """
 import random
 import pytest
-from . import utils
+from tests import utils
```

Comment:
This is still a relative import, but I'm not sure if we can avoid that here since it's not a module...